src/Microsoft.Win32.Registry/tests/RegistryKey/RegistryKey_CreateSubKey_str.cs

```diff
@@ -53,6 +41,22 @@ public void NegativeTests()
             });
         }
 
+        [Fact(Skip = "This test fails on Win10 Anniversary Update Issue #10546")]
```

Comment:
I don't really see the value in this part of the test personally- I'll leave it to @ianhays to make the final decision with the original issue.